src/test/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImplTest.java

```diff
@@ -461,4 +468,46 @@ private void assertExpectedOrganisationResponse(OrganisationResponse organisatio
         assertThat(organisationResponse.getOrganisationIdentifier()).isNotNull();
         assertThat(organisationResponse.getOrganisationIdentifier().length()).isEqualTo(orgIdLength);
     }
+
+    @Test
+    public void testDeletePendingOrganisation() {
+
+        ProfessionalUser profile = new ProfessionalUser("firstName", "lastName", "email@org.com", organisation);
+        superUser.setUserIdentifier(UUID.randomUUID().toString());
+        List<SuperUser> users = new ArrayList<>();
+        users.add(superUser);
+
+        organisation.setStatus(OrganisationStatus.PENDING);
+        organisation.setUsers(users);
+
+        contactInformation.setAddressLine1("addressLine1");
+
+        List<DxAddress> dxAddresses = new ArrayList<>();
+        List<ContactInformation> contactInformations = new ArrayList<>();
+
+        dxAddresses.add(dxAddress);
+        contactInformation.setDxAddresses(dxAddresses);
+        contactInformations.add(contactInformation);
+        organisation.setContactInformations(contactInformations);
+
+        PrdEnum prdEnum = new PrdEnum(new PrdEnumId(0, "SIDAM_ROLE"), "pui-user-manager", "SIDAM_ROLE");
+        UserAttribute userAttribute = new UserAttribute(professionalUser,prdEnum);
+        userAttributes.add(userAttribute);
+        professionalUser.setUserAttributes(userAttributes);
+
+        UserAccountMapId userAccountMapId = new UserAccountMapId(professionalUser, paymentAccount);
+        UserAccountMap userAccountMap = new UserAccountMap(userAccountMapId);
+        userAccountMaps.add(userAccountMap);
+        paymentAccount.setUserAccountMap(userAccountMaps);
+        organisation.addPaymentAccount(paymentAccount);
+
+        when(organisationRepository.findByOrganisationIdentifier(organisationIdentifier)).thenReturn(organisation);
```

Comment:
is it possible to add a `verify()` for this `when()`?

Reply:
when is not required here so removed the entry